docs/index.md

```diff
@@ -17,16 +17,24 @@ Pebble is useful for developers who are building [Juju charms on Kubernetes](htt
 
 ```{grid-item-card} [How-to guides](how-to/index)
 **Step-by-step guides** covering key operations and common tasks.
+- [Install Pebble](how-to/install-pebble)
+- [Manage service dependencies](how-to/service-dependencies)
+- [Manage identities](how-to/manage-identities)
```

Comment:
Side note: do you know how we configure the ordering of these items in the left-hand sidebar menu? I would have thought they'd be ordered alphabetically. (Here I think ordering by importance makes sense.)

Reply:
It depends on the order of the `toctree` defined in `docs/how-to/index.md`.

It makes more sense to order by importance. Which order shall we use? I think:

```yaml
# How-to guides

## Service orchestration
## Identities
## Installation
```

Is this OK? However, it seems a little odd on the how-to index page because normally you wouldn't finish an article with how to install something...

Reply:
Per discussion, I think the current order is fine, actually -- I was just wondering why it wasn't alphabetical. But the manual ordering here is fine as is. I like "Install Pebble" first.